package.json

```diff
@@ -126,7 +126,7 @@
     "browserify": "16.5.1",
     "cross-env": "5.2.0",
```

Comment:
do we still need this dependency?

Reply:
I think we needed this for Windows build. Will investigate and remove if not required